docs/src/howto-guides/010_add_self_signed_trusted.md

```diff
@@ -0,0 +1,43 @@
+# How to add self-signed certificate to trusted certificates list
+
+## Overview
+
+For secure connectivity to work you need to add CA certificate to the trusted certificate list repository.
```

Comment:
"For secure connectivity to work you need to add CA certificate to the trusted certificate list repository" - is only true when the server is using a certificate with an untrusted root.